cms/djangoapps/contentstore/views/tests/test_library.py

```diff
@@ -84,21 +84,6 @@ def test_create_library(self):
         self.assertEqual(response.status_code, 200)
         # That's all we check. More detailed tests are in contentstore.tests.test_libraries...
 
-    @patch.dict('django.conf.settings.FEATURES', {'ENABLE_CREATOR_GROUP': True})
-    def test_lib_create_permission(self):
-        """
-        Users who aren't given course creator roles shouldn't be able to create
-        libraries either.
-        """
-        self.client.logout()
-        ns_user, password = self.create_non_staff_user()
-        self.client.login(username=ns_user.username, password=password)
-
-        response = self.client.ajax_post(LIBRARY_REST_URL, {
-            'org': 'org', 'library': 'lib', 'display_name': "New Library",
-        })
-        self.assertEqual(response.status_code, 403)
```

Comment:
@bradenmacdonald same here - it would be better if test checked for successful library creation.